package.json

```diff
@@ -69,7 +69,7 @@
     "@storybook/node-logger": "^6.2.9",
     "@storybook/preset-create-react-app": "^3.1.7",
     "@storybook/react": "^6.2.9",
-    "babel-loader": "^8.2.2",
+    "babel-loader": "^8.1.0",
```

Comment:
Any specific reason to downgrade?

Reply:
got some dependencies error for that version,  but later i realized the problem is in another package . sorry, forgot to revert it back. changing the version as it was before.